payment/src/test/resources/payment.properties

```diff
@@ -1,3 +1,4 @@
 org.killbill.payment.failure.retry.start.sec=3600
 org.killbill.payment.failure.retry.multiplier=1
 org.killbill.payment.failure.retry.max.attempts=3
+org.killbill.payment.plugin.timeout=5s
```

Comment:
Reduce it to `1s`.

Also, make sure to re-run all tests (mysql and travis profiles) to see if there is no side effect.